040-SDK/130-transaction.mdx

```diff
@@ -343,15 +408,179 @@ result, _ := recordsClient.Transaction(context.TODO(), xata.TransactionRequest{
+      Table: "titles",
+      Where: map[string]any{
+        "id": xata.String("rec_cfl4g6v00056"),
+    },
+  }),
+   // Additional transactions can be added here
+  ,
+})
+```
+
+```jsonc
+// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/transaction
+{
+  "operations": [
+    {
+      "delete": {
+        "table": "titles",
+        "where": { "id": "rec_cfl4g6v00023" }
+      }
+    },
+    {
+      "delete": {
+        "table": "titles",
+        "where": { "id": "rec_cfl4g6v00056" }
+      }
+    }
+    // Additional transactions can be added here
+  ]
+}
+```
+
+</TabbedCode>
+
 ### gets
 
-A get is used to retrieve a record by id. A get operation can retrieve records created in the same transaction but will not cancel
+Get is used to retrieve a record by ID. A get operation can retrieve records created in the same transaction but will not cancel
 a transaction if no record is found.
 
+<TabbedCode tabs={['TypeScript', 'Python', 'Go', 'JSON']}>
+
+```ts
+const result = await xata.transactions.run([
+  {
+    get: {
+      table: 'titles',
+      where: { id: 'rec_cfl4g6v00023' }
+    }
+  },
+  {
+    get: {
+      table: 'titles',
+      where: { id: 'rec_cfl4g6v00056' }
+    }
+  }
+]);
+```
+
+```python
+result = xata.records().transaction({
+  "operations": [
+    {
+      "get": {
+        "table": "titles",
+        "where": { "id": "rec_cfl4g6v00023" }
+      }
+    },
+    {
+      "get": {
+        "table": "titles",
+        "where": { "id": "rec_cfl4g6v00056" }
+      }
+    }
+  ]
+})
+```
+
+```go
+recordsClient, _ := xata.NewRecordsClient()
+result, _ := recordsClient.Transaction(context.TODO(), xata.TransactionRequest{
+  Operations: []xata.TransactionOperation{
+    xata.NewGetTransaction(xata.TransactionGetOp{
+      Table: "titles",
+      Where: map[string]any{
+        "id": xata.String("rec_cfl4g6v00023"),
+      },
+    }),
+    xata.NewGetTransaction(xata.TransactionGetOp{
+      Table: "titles",
+      Where: map[string]any{
+        "id": xata.String("rec_cfl4g6v00056"),
+      },
+    }),
+  },
+})
+```
+
+```jsonc
+// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/transaction
+{
+  "operations": [
+    {
+      "get": {
+        "table": "titles",
+        "where": { "id": "rec_cfl4g6v00023" }
+      }
+    },
+    {
+      "get": {
+        "table": "titles",
+        "where": { "id": "rec_cfl4g6v00056" }
+      }
+    }
+  ]
+}
+```
+
+</TabbedCode>
+
 ## Limitations
 
 - A maximum of 1000 operations are allowed in a single transaction. If you exceed this limit, you will receive an error message.
-- Inserting and updating content to file and file\[] (file array) columns is not permitted with transactions.
+- Inserting and updating content to `file` and `file\[]` (file array) columns is not permitted with transactions.
+- To avoid the SQL proxy timeout, delete in small batches. It is recommended to delete no more than 10,000 records per batch.
```

Comment:
```suggestion
```

I suggest we remove this from the transactions page as SQL DELETE is not a transaction and it might confuse users to find it written here